### ISSUE TITLE
Invalid regex escape sequence

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#1438]
Not sure what's going on here. Celery logs are reporting when celery starts, that:

task-server_celery_prefork.1.8dqfwhb2syb1@debian    | /opt/celery/cl/lib/string_diff.py:12: DeprecationWarning: invalid escape sequence \.
task-server_celery_prefork.1.8dqfwhb2syb1@debian    |   "a|an|and|as|at|but|by|en|etc|for|if|in|is|of|on|or|the|to|v\.?|via"
task-server_celery_prefork.1.8dqfwhb2syb1@debian    | /opt/celery/cl/lib/string_diff.py:13: DeprecationWarning: invalid escape sequence \.
task-server_celery_prefork.1.8dqfwhb2syb1@debian    |   + "|vs\.?|united|states?|et|al|appellants?|defendants?|administrator|plaintiffs?|error"

Something to look into. Should be easy.

**Comment by tirkarthi** [comment]
Full list as below : 

```
find . -iname '*.py' | xargs -P4 -I{} python3 -Wall -m py_compile {}     
./cl/lib/string_diff.py:12: DeprecationWarning: invalid escape sequence \.
  "a|an|and|as|at|but|by|en|etc|for|if|in|is|of|on|or|the|to|v\.?|via"
./cl/lib/string_diff.py:13: DeprecationWarning: invalid escape sequence \.
  + "|vs\.?|united|states?|et|al|appellants?|defendants?|administrator|plaintiffs?|error"
./cl/lib/pacer.py:302: DeprecationWarning: invalid escape sequence \(
  phone = re.sub("(\(|\)|\s+)", "", phone)
./cl/lib/pacer.py:414: DeprecationWarning: invalid escape sequence \s
  line = re.sub("Email:\s*", "", line).strip()
./cl/people_db/import_judges/populate_fjc_judges.py:48: DeprecationWarning: invalid escape sequence \,
  else re.split("\,+\s+(?=\d)+|\,+\s+(?=\-)", a, 1)
./cl/people_db/import_judges/populate_fjc_judges.py:145: DeprecationWarning: invalid escape sequence \,
  else re.split("\,+\s+(?=\d)+", a, 1)
./cl/people_db/import_judges/populate_fjc_judges.py:148: DeprecationWarning: invalid escape sequence \s
  ",+\s+(?=June|March|January|February|April|May|July|August|September|October|November|December|Fall|Spring)+",
./cl/corpus_importer/import_columbia/parse_judges.py:281: DeprecationWarning: invalid escape sequence \.
  first_m_last = "%s [a-z]\.? %s" % (names[i - 1], names[i])
./cl/corpus_importer/import_columbia/parse_judges.py:305: DeprecationWarning: invalid escape sequence \.
  first_m_last = "%s [a-z]\.? %s" % judge
./cl/corpus_importer/import_columbia/html_test.py:112: DeprecationWarning: invalid escape sequence \s
  bad_punctuation = "(-|–|_|/|;|,|\s)*"
./cl/corpus_importer/import_columbia/html_test.py:132: DeprecationWarning: invalid escape sequence \.
  US = "USA|U\.S\.A\.|U\.S\.?|U\. S\.?|(The )?United States of America|The United States"
./cl/corpus_importer/import_columbia/html_test.py:135: DeprecationWarning: invalid escape sequence \s
  ET_AL = re.compile(",?\set\.?\sal\.?", re.I)
./cl/corpus_importer/import_columbia/html_test.py:138: DeprecationWarning: invalid escape sequence \.
  + "|devisee|executor|executrix|pet(\.|itioner)s?|petitioners?(--?|/)appell(ee|ant)s?"
./cl/corpus_importer/import_columbia/html_test.py:149: DeprecationWarning: invalid escape sequence \.
  SMALL = "a|an|and|as|at|but|by|en|for|if|in|is|of|on|or|the|to|v\.?|via|vs\.?"
./cl/corpus_importer/import_columbia/html_test.py:631: DeprecationWarning: invalid escape sequence \s
  raw_parts = re.split("(?<=[0-9][0-9][0-9][0-9])(\s|.)", raw_date)
./cl/corpus_importer/import_columbia/html_test.py:661: DeprecationWarning: invalid escape sequence \d
  text = re.compile("(\d+)").split(raw_part.lower())[0].strip()
./cl/corpus_importer/import_columbia/html_test.py:665: DeprecationWarning: invalid escape sequence \[
  text = re.sub("(\[fn.?\])", "", text)
./cl/corpus_importer/import_columbia/parse_opinions.py:336: DeprecationWarning: invalid escape sequence \s
  raw_parts = re.split("(?<=[0-9][0-9][0-9][0-9])(\s|.)", raw_date)
./cl/corpus_importer/import_columbia/parse_opinions.py:359: DeprecationWarning: invalid escape sequence \d
  text = re.compile("(\d+)").split(raw_part.lower())[0].strip()
./cl/corpus_importer/import_columbia/parse_opinions.py:363: DeprecationWarning: invalid escape sequence \[
  text = re.sub("(\[fn.?\])", "", text)
./cl/corpus_importer/import_columbia/convert_columbia_html.py:40: DeprecationWarning: invalid escape sequence \*
  fnum = re.search("[\*\d]+", ref).group()
./cl/corpus_importer/import_columbia/convert_columbia_html.py:42: DeprecationWarning: invalid escape sequence \[
  fnum = re.search("\[fn(.+)\]", ref).group(1)
./cl/corpus_importer/import_columbia/convert_columbia_html.py:54: DeprecationWarning: invalid escape sequence \*
  fnum = re.search("[\*\d]+", ref).group()
./cl/corpus_importer/import_columbia/convert_columbia_html.py:56: DeprecationWarning: invalid escape sequence \[
  fnum = re.search("\[fn(.+)\]", ref).group(1)
./cl/corpus_importer/import_columbia/convert_columbia_html.py:68: DeprecationWarning: invalid escape sequence \s
  text = re.sub("</blockquote>\s*<blockquote>", "\n\n", text)
./cl/corpus_importer/import_columbia/convert_columbia_html.py:70: DeprecationWarning: invalid escape sequence \s
  text = re.sub("<p>\s*<blockquote>", "<blockquote><p>", text, re.M)
./cl/corpus_importer/import_columbia/regexes_columbia.py:19: DeprecationWarning: invalid escape sequence \.
  (re.compile("U\.S\. Circuit Court", re.I), "circtnc"),
./cl/corpus_importer/import_columbia/analyze_import_exceptions.py:46: DeprecationWarning: invalid escape sequence \d
  newseg = re.sub("\d", "#", seg)
./cl/search/constants.py:38: DeprecationWarning: invalid escape sequence \s
  """
./cl/search/forms.py:552: DeprecationWarning: invalid escape sequence \|
  q = re.sub("\|", " OR ", q)
./cl/citations/reporter_tokenizer.py:56: DeprecationWarning: invalid escape sequence \d
  if re.match("\d+\-[A-Za-z]+\-\d+", text):
./cl/custom_filters/templatetags/text_filters.py:68: DeprecationWarning: invalid escape sequence \s
  return mark_safe(re.sub("\s", "&nbsp;", esc(text.strip())))
./cl/custom_filters/templatetags/text_filters.py:80: DeprecationWarning: invalid escape sequence \.
  re.sub(" v\. ", '<span class="alt"> v. </span>', esc(text))
./cl/simple_pages/views.py:256: DeprecationWarning: invalid escape sequence \d
  if re.match("\d{3}", cd["phone_number"]):
./cl/recap/management/commands/pacer_iquery_scraper.py:59: DeprecationWarning: invalid escape sequence \.
  "^https://www\.courtlistener\.com/docket/([0-9]+)/", url
./cl/cleanup/tests.py:366: DeprecationWarning: invalid escape sequence \F
  """
```

**Comment by tirkarthi** [comment]
pyupgrade can help with these warnings : https://github.com/asottile/pyupgrade#invalid-escape-sequences

**Comment by mlissner** [comment]
Hoo boy. Nice one, @tirkarthi, thank you for sending this. Are you interested in doing more to help out?

**Comment by mlissner** [comment]
Closed via #1467.
